LambdaRuntimeDockerfiles/update-dockerfile.ps1

```diff
@@ -0,0 +1,72 @@
+param ([Parameter(Mandatory)]$Dockerfile)
+
+# Updates the Dockerfile with next ASP.NET Core version and checksum256 hash if available
+function Update-Dockerfile ([string]$path) {
```

Comment:
Can you add some documentation on how this process works and what it can and can't do?

If I'm correctly reading `Update-Dockerfile` -> `Get-NextASPNETVersion` -> `Update-PatchVersion`, then won't this script break when Microsoft updates a Minor version?  Is that by design?

Reply:
Added top-level docs about the process.

That's correct, it will break in case of a minor version but Microsoft will never (in theory they can) release a minor version based on the history of updates. If they do, then there will be a complete different conversion on supporting new minor version, may be a new dockerfile etc etc. In short, yes script is designed to update only patch version.

Reply:
> Microsoft will never ... release a minor version based on the history of updates:

I think there's pretty strong historical evidence that Microsoft does release minor updates:

.NET Core 3.1
.NET Core 3.0
.NET Core 2.2
.NET Core 2.1
.NET Core 2.0
.NET Core 1.1
.NET Core 1.0
.NET Framework 4.8
.NET Framework 4.7
.NET Framework 4.6
.NET Framework 4.5
.NET Framework 4
.NET Framework 3.5
.NET Framework 3.0

Reply:
Not after they release a stable version, like 3.1, 2.1 (we don't support 2.2). If we ever do, we have to make necessary change like mentioned, it will be a new dockerfile.